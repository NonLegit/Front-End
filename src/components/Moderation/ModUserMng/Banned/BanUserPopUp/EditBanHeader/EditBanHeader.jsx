import { Typography } from '@mui/material';
import CloseIcon from '@mui/icons-material/Close';
import StyledBox from './styles';

/**
 * EditBan popup header
 * @component
 * @property  {function} handleClickCloseBan close the popup
 * @property  {string} userName the username of banned user
 * @return {React.Component} - EditBan popup header
 */

function EditBanHaeder(props) {
  const { handleClickCloseBan, userName } = props;
  return (
    <StyledBox>
      <Typography
        fontWeight={600}
      >
        Edit ban for: u/
        {userName}
      </Typography>
      <CloseIcon
        sx={{ cursor: 'pointer' }}
        onClick={handleClickCloseBan}
      />
    </StyledBox>
  );
}

export default EditBanHaeder;
