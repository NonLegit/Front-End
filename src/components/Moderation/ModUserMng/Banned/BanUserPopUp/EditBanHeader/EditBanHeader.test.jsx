import renderer from 'react-test-renderer';
import EditBanHeader from './EditBanHeader';

test('test snapshot', async () => {
  const tree = renderer.create(<EditBanHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});
