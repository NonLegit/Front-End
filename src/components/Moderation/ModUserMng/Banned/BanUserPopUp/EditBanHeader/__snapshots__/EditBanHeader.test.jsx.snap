// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test snapshot 1`] = `
<div
  className="MuiBox-root css-990j2o"
>
  <p
    className="MuiTypography-root MuiTypography-body1 css-1qqynwt-MuiTypography-root"
  >
    Edit ban for: u/
  </p>
  <svg
    aria-hidden={true}
    className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-lw08pq-MuiSvgIcon-root"
    data-testid="CloseIcon"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <path
      d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
    />
  </svg>
</div>
`;
